osu.Game/Screens/Play/Player.cs

```diff
@@ -184,7 +184,7 @@ private void load(AudioManager audio, OsuConfigManager config)
             foreach (var mod in Mods.Value.OfType<IApplicableToHealthProcessor>())
                 mod.ApplyToHealthProcessor(HealthProcessor);
 
-            BreakOverlay.IsBreakTime.ValueChanged += _ => updatePauseOnFocusLostState();
+            BreakOverlay.IsBreakTime.BindValueChanged(_ => onBreakTimeChanged(), true);
```

Comment:
Doesn't really make sense to discard the `ValueChangedEvent<bool>` argument if there's a read of `IsBreakTime.Value` in `onBreakTimeChanged`. Even more, I'd argue it's incorrect as it's possibly race-prone.